src/main/webapp/activityFeed.jsp

```diff
@@ -0,0 +1,71 @@
+<%--
+ 
+  Modeled after conversations.jsp
```

Comment:
I think we should add new pages to the WEB-INF/view/ directory (like the registration page in Project 1), rather than the top-level webapp directory.